system/mrm_handler/config/mrm_handler.param.yaml

```diff
@@ -4,6 +4,8 @@
   ros__parameters:
     update_rate: 10
     timeout_operation_mode_availability: 0.5
+    timeout_call_mrm_behavior: 0.01
+    timeout_cancel_mrm_behavior: 0.01
```

Comment:
Isn't it too small?
ROS 2 executor sometimes takes more than 10 milliseconds for a normal (successful) service operation.

Reply:
It bothers me.
I have not been able to do a proper investigation to determine how much timeout should be.
In my tests with psim, even `timeout_call_mrm_behavior: 0.001` worked fine.
What are your thoughts on this?

Reply:
> In my tests with psim, even timeout_call_mrm_behavior: 0.001 worked fine.
It sounds strange for me.
In my past personal mesurements on latencies between two nodes, it was usually a few milliseconds.

Reply:
The timeout value is left unchanged.